### DISCUSSION TITLE
Help deciding on Hardware - Jetson - J4125 - 3610QM

### DISCUSSION THREAD

**Discussion by sibero80** [blakeblackshear/frigate#2265] [Ask A Question]
Greetings from Colombia.
I wanted your input regarding hardware selection, given the supply and tax challenges in my country.
I need to buy the best device money can buy for less than 200 (199), this way I dont get taxed by imports that can double the cost.

**Options are:**
- Jetson Nano 4GB
- Mini PC with J4125
- Second Hand Laptop with i7-3210QM and GeForce 610M / 2GB DDR3 VRAM

**Concerns:**
I was able to procure a Coral M.2 Key E . Dual TPU.
I know there are Key E to Mini PCIe but I have not seen any Key E to Key B+M, that most of the MiniPC uses.
Jetson Nano is attractive for the "future proofing" in advances on GPU offloading in ARM, but I have not seen yet a succesful frigate installation on this hardware.
So far laptop seems the obvious choice, but I'm concerned about power consumption that can be 4x to 5x of the other options.

Do you know any good J4125 mini PC with MiniPCIe or M2 Key 2 slots?  or any suggestions of M2 Key B+M adapters to Key E?
All comments and suggestions greatly appreciated!

**Comment by sibero80** [comment]
To who it may concern, I ended up going for a J4125 board with M.2 Key-E slot.
It was a nightmare to find under 200USD and that shipped to my country.

https://www.aliexpress.com/item/1005003110653216.html?spm=a2g0s.9042311.0.0.fa8a4c4dylS4Dg

The one with M.2 the Key-E is the 2x HDMI version.

Will report back once I have it in my hands.

**Reply by eggnog-armed** [reply]
Hi sibero80!

Did you have success with 2x HDMI board at that link and your Coral M.2 Key E . Dual TPU?

thanks
